summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -179,10 +173,21 @@ abstract class Storm(options: Map[String, Options], transformConfig: Summingbird
       }
     }
 
-    PlannedTopology(config, StormTopologyBuilder(options, jobId, stormDag).build)
+    val topology = StormTopologyBuilder(options, jobId, stormDag).build
+    PlannedTopology(config, topology.build(jobId))
   }
   def run(tail: TailProducer[Storm, _], jobName: String): Unit = run(plan(tail), jobName)
   def run(plannedTopology: PlannedTopology, jobName: String): Unit
+
+  private[storm] def planDag[T](tail: TailProducer[Storm, T]): Dag[Storm] = {
+    /*
+     * TODO: storm does not yet know about ValueFlatMapped, so remove it before
```

Comment:
Actually this is not needed anymore, removed this code and added test for `mapValues` into `StormLaws`. Thanks!